app/components/message_attachments/message_attachment.js

```diff
@@ -385,23 +386,6 @@ export default class MessageAttachment extends PureComponent {
 
         let text;
         if (attachment.text) {
-            let moreLessLocale = {id: 'post_attachment.collapse', defaultMessage: 'Show less...'};
-            if (collapsed) {
-                moreLessLocale = {id: 'post_attachment.more', defaultMessage: 'Show more...'};
```

Comment:
Separate PR will be submitted to webapp to remove the following translations:
- `post_attachment.collapse`
- `post_attachment.more`